docs/src/userguide/language_basics.rst

```diff
@@ -119,40 +187,88 @@ You can read more about them in :ref:`extension-types`.
 Types
 -----
 
-Cython uses the normal C syntax for C types, including pointers.  It provides
-all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
-``long long`` as well as their ``unsigned`` versions, e.g. ``unsigned int``.
-The special ``bint`` type is used for C boolean values (``int`` with 0/non-0
-values for False/True) and ``Py_ssize_t`` for (signed) sizes of Python
-containers.
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        Cython supports annotaions for C types, including pointers.  It provides
+        all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
+        ``long long`` as well as their ``unsigned`` versions, e.g. ``uint``.
+        The special ``bint`` type is used for C boolean values (``int`` with 0/non-0
+        values for False/True) and ``Py_ssize_t`` for (signed) sizes of Python
+        containers.
+
+        Pointer types are constructed by appending a ``p_`` to the base type
+        they point to, e.g. ``pp_int`` for a pointer to a pointer to a C int.
+        Further pointer types can be constructed with ``pointer()`` function.
+        Arrays use the normal C array syntax, e.g. ``int[10]``, and the size must be known
+        at compile time for stack allocated arrays. Cython doesn't support variable length arrays from C99.
+        Note that Cython uses array access for pointer dereferencing.
+
+    .. group-tab:: Cython
 
-Pointer types are constructed as in C, by appending a ``*`` to the base type
-they point to, e.g. ``int**`` for a pointer to a pointer to a C int.
-Arrays use the normal C array syntax, e.g. ``int[10]``, and the size must be known
-at compile time for stack allocated arrays. Cython doesn't support variable length arrays from C99.
-Note that Cython uses array access for pointer dereferencing, as ``*x`` is not valid Python syntax,
-whereas ``x[0]`` is.
+        Cython language uses the normal C syntax for C types, including pointers.  It provides
+        all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
+        ``long long`` as well as their ``unsigned`` versions, e.g. ``unsigned int``.
+        The special ``bint`` type is used for C boolean values (``int`` with 0/non-0
+        values for False/True) and ``Py_ssize_t`` for (signed) sizes of Python
+        containers.
+
+        Pointer types are constructed as in C, by appending a ``*`` to the base type
+        they point to, e.g. ``int**`` for a pointer to a pointer to a C int.
+        Arrays use the normal C array syntax, e.g. ``int[10]``, and the size must be known
+        at compile time for stack allocated arrays. Cython doesn't support variable length arrays from C99.
+        Note that Cython uses array access for pointer dereferencing, as ``*x`` is not valid Python syntax,
+        whereas ``x[0]`` is.
```

Comment:
I don't get this addiction, can you explain what do you mean by this, please?
", as ``*x`` is not ..."

Reply:
The sentence seems correct and clear to me.